Makefile

```diff
@@ -19,6 +19,9 @@ TESTS =
 
 OLDTARGETS = linux win bsd osx sun
 
+ifeq ($(COLORS),0)
```

Comment:
Done with https://github.com/woodruffw/screenfetch-c/pull/32/commits/2106e95a827bd09982facd57a1025e553cee4750

By the way, I think you can change the line `Ubuntu (tested with Lubuntu)` in the README to `Ubuntu`. I'm using the regular Ubuntu and it works fine for me.